controllers/quote.js

```diff
@@ -121,16 +122,23 @@ const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
 /* eslint-enable */
 
 exports.get = function *() {
-  let quote = yield backend.getQuote({
-    source_ledger: this.query.source_ledger,
-    destination_ledger: this.query.destination_ledger,
-    source_amount: this.query.source_amount,
-    destination_amount: this.query.destination_amount
-  })
+  const query = new QuoteQuery(this.query)
```

Comment:
It isn't a conflict; feature/dj-quote-balance is based on feature/dj-payment-template